src/fs/file_format.c

```diff
@@ -14,6 +14,10 @@
 enum fformat raw_get_file_format(uint8_t *file) {
 	/* signature of PNG files */
 	uint8_t png_signature[8] = {0x89, 0x50, 0x4e, 0x47, 0x0d, 0x0a, 0x1a, 0x0a};
+	/* signature of ELF files */
+	uint8_t elf_signature[4] = {0x7f, 0x45, 0x4c, 0x46};
```

Comment:
Please use readable characters when it's meaningful (i.e. `'E'` instead of `0x45`), this way it's easier to see what's going on.

Reply:
Thanks for the review, I'll make the required changes